### DISCUSSION TITLE
I/O URL actions support

### DISCUSSION THREAD

**Discussion by Snekatis** [mongoose-os-apps/shelly-homekit#1484] [Ideas]
Can you add support for I/O URL actions like in factory fw?

<img width="666" alt="s" src="https://user-images.githubusercontent.com/459219/184608930-f215340c-08fe-4415-8a65-0ec481ff7a73.png">

**Comment by timoschilling** [comment]
That's something you can solve with HomeKit automatisations.

**Comment by Snekatis** [comment]
Yes, it can be solved this way, but unfortunately the Homekit automation does not respond immediately (delay 2s-4s) and you cannot influence the settings like a long press.

**Comment by rojer** [comment]
i guess we could accommodate this and add at least some REST API compatible with stock fw.
shouldn't be too difficult.

**Comment by tob1no** [comment]
Hi, is there any news on this? I'd be interested aswell.
